src/Core/Jobs/BaseJobsHostedService.cs

```diff
@@ -18,25 +18,45 @@ public abstract class BaseJobsHostedService : IHostedService, IDisposable
         protected readonly ILogger _logger;
 
         private IScheduler _scheduler;
+        protected GlobalSettings _globalSettings;
 
         public BaseJobsHostedService(
+            GlobalSettings globalSettings,
             IServiceProvider serviceProvider,
             ILogger logger,
             ILogger<JobListener> listenerLogger)
         {
             _serviceProvider = serviceProvider;
             _logger = logger;
             _listenerLogger = listenerLogger;
+            _globalSettings = globalSettings;
         }
 
         public IEnumerable<Tuple<Type, ITrigger>> Jobs { get; protected set; }
 
         public virtual async Task StartAsync(CancellationToken cancellationToken)
         {
-            var factory = new StdSchedulerFactory(new NameValueCollection
+            var props = new NameValueCollection
             {
-                { "quartz.serializer.type", "binary" }
-            });
+                {"quartz.serializer.type", "binary"},
+            };
+
+            if (!_globalSettings.SelfHosted && !string.IsNullOrEmpty(_globalSettings.SqlServer.JobSchedulerConnectionString))
```

Comment:
we're not allowing this for self-hosted people?

Reply:
I suppose we could, that wasn't the direction we (or at least I had) given Oscar, I'm good to simply key off of the connection string settings and remove the SelfHosted bit. Sorry @Hinton , that was my bad.

Reply:
As long as we aren't setting this connection string in our self hosted config, I don't see a reason to stop them (a self-hosted user) if they wanted to.